src/main/java/uk/gov/hmcts/juror/api/moj/repository/AppearanceRepository.java

```diff
@@ -75,7 +75,11 @@ List<Appearance> findAllByCourtLocationLocCodeAndJurorNumberAndAttendanceDateInO
     @Query(value = "select nextval('juror_mod.attendance_audit_seq')", nativeQuery = true)
     Long getNextAttendanceAuditNumber();
 
-    long countByJurorNumberAndAppearanceStageNotNull(String jurorNumber);
```

Comment:
I think the query needs to look like

select count(*) from juror_mod.appearance a where a.juror_number=  ?1 and a.appearance_stage is not null and( a.attendance_type is null or a.attendance_type not in ('ABSENT'))

Reply:
This would be true if we had the native query flag set to true :)
But as we are using spring jpa we use the class variable names instead of the database table column names (Makes it type safe in Java)

Reply:
ok, didnt notice that :-)